src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -772,4 +773,30 @@ BOOST_AUTO_TEST_CASE(HardShrinkFunctionTest)
                                    desiredDerivatives);
 }
 
+/**
+ * Basic test of the EliSH function.
+ */
+BOOST_AUTO_TEST_CASE(ElishFunctionTest)
+{
+  const arma::colvec desiredActivations("-0.10307056080762242 \
```

Comment:
Are those numbers manually calculated or via a another framework?

Reply:
Thanks @zoq for such excellent review. I haven't used any framework for the calculation, instead, used basic NumPy & python.

Reply:
Okay, do you mind to add a comment, that points out this values are manually calculated?

Reply:
Added.